tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeChecker.scala

```diff
@@ -242,6 +242,18 @@ class PluginTypeChecker extends Logging {
     score
   }
 
+  def getWriteFormatString(node: String): String = {
+    // String for Write data format will be in below format and we need to parse the string to
+    // fetch Parquet(which is the write format in below example).
+    // Execute InsertIntoHadoopFsRelationCommand
+    // file:/home/ubuntu/spark-3.1.1-eventlogs/complex_nested_decimal,
+    // false, Parquet, Map(path -> complex_nested_decimal)
```

Comment:
honestly I'm not following this comment and it doesn't seem to match the text... ie what I saw was:

`Execute InsertIntoHadoopFsRelationCommand InsertIntoHadoopFsRelationCommand somefile://8e9/, false, [va1#25, var2#26], ORC, Map(path`


The issue was the "[va1#25, var2#26]"  part. I don't see that in the example or where that fits in... so can we update the comment.  Does the Map always show up?  even if its empty if that is possible.   I'll have to go look at Spark code that prints this.

Reply:
I have updated the comment.

> Does the Map always show up?
    That's a good point. I ran the query on different Spark versions. `Map` is present in Spark-3.1 and Spark-3.2. It will have atleast one key->value pair of `path-> actualPath` . However, we don't see `Map` in the eventlogs of Spark-3.3 and Spark-3.4-SNAPSHOT. Rather it's in this format -  `[ key->value ]`.  I have updated the code to get the write format correctly for all versions. Have included this in tests as well. PTAL